node/cn/peer_set_test.go

```diff
@@ -524,6 +524,129 @@ func TestPeerSet_SampleResendPeersByType_EN(t *testing.T) {
 	}
 }
 
+func TestPeerSet_SampleResendPeersByType_Default(t *testing.T) {
+	peerSet := newPeerSet()
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	cnPeer1 := NewMockPeer(mockCtrl)
+	cnPeer2 := NewMockPeer(mockCtrl)
+	pnPeer1 := NewMockPeer(mockCtrl)
+	pnPeer2 := NewMockPeer(mockCtrl)
+	enPeer1 := NewMockPeer(mockCtrl)
+	enPeer2 := NewMockPeer(mockCtrl)
+
+	setMockPeersConnType(cnPeer1, pnPeer1, enPeer1)
+	setMockPeersConnType(cnPeer2, pnPeer2, enPeer2)
+	setMockPeers([]*MockPeer{cnPeer1, pnPeer1, enPeer1, cnPeer2, pnPeer2, enPeer2})
+
+	assert.NoError(t, peerSet.Register(cnPeer1))
+	assert.NoError(t, peerSet.Register(pnPeer1))
+	assert.NoError(t, peerSet.Register(enPeer1))
+
+	assert.NoError(t, peerSet.Register(cnPeer2))
+	assert.NoError(t, peerSet.Register(pnPeer2))
+	assert.NoError(t, peerSet.Register(enPeer2))
+
+	assert.Nil(t, peerSet.SampleResendPeersByType(node.UNKNOWNNODE))
```

Comment:
Minor comment:
I think that it will be better if SampleResendPeersByType() returns an error and check the error for the default case.